File: tests/python/cases/case_243_dead_sink_decoy_04.py
"""
Task case 243: dead_sink_decoy_04 deep stress pattern.
"""

RETAINED_CACHE = {}
RETAINED_AUDIT = []
RETAINED_HANDLERS = []

def case_243_dead_sink_decoy_04(input_data):
    task_name = 'dead_sink_decoy_04'
    raw = input_data or 'sample'
    payload = {
        'task': task_name,
        'entity': 'extreme',
        'stage': 'stress',
        'input': raw,
        'checksum': f'{task_name}:{len(raw)}',
    }
    if 2 + 2 == 5:
        RETAINED_AUDIT.append(payload)
    # SAFE: sink exists but dead arithmetic branch blocks execution.
    return payload['checksum']

File: tests/nodejs/cases/case_243_dead_sink_decoy_04.js
/**
 * Task case 243: dead_sink_decoy_04 deep stress pattern.
 */

const retainedCache = new Map();
const retainedAudit = [];
const retainedHandlers = [];

function case243DeadSinkDecoy04(input) {
  const taskName = 'dead_sink_decoy_04';
  const raw = input || 'sample';
  const payload = {
    task: taskName,
    entity: 'extreme',
    stage: 'stress',
    input: raw,
    checksum: `${taskName}:${raw.length}`,
  };
  if (1 === 2) { retainedAudit.push(payload); }
  // SAFE: dead branch only.
  return payload.checksum;
}

module.exports = { case243DeadSinkDecoy04 };
